docs/overview/getting-started/virtual-machines.md

```diff
@@ -0,0 +1,94 @@
+# Virtual Machines
+
+Avalanche allows developers to implement their Dapp in the language of their choice using the 
+Virtual Machine (VM) framework. VMs define blockchains, and serve as application-level blueprints 
+for how blockchains are created.
+
+## Validators
+
+All Avalanche validators as members of the Avalanche primary network are required to run three VMs:
+
+- Coreth: Defines the Contract Chain (C-Chain); supports smart contracts written in Solidity.
```

Comment:
You need to add the words EVM-compatible. Technically it supports languages other than Solidity too